Assignment4/carsanvel-e22a349/task09.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Task09.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1MbnICFbdB52h0b3jqQPm6kWW4GJEs3N8
+
+**Task 09: Data linking**
+"""
+
+# !pip install rdflib
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials/"
+
+from rdflib import Graph, Namespace, Literal, URIRef
+from rdflib.namespace import RDF, RDFS, OWL
+from rdflib.plugins.sparql import prepareQuery
+g1 = Graph()
+g2 = Graph()
+g3 = Graph()
+g1.parse(github_storage+"rdf/data03.rdf", format="xml")
+g2.parse(github_storage+"rdf/data04.rdf", format="xml")
+
+"""Busca individuos en los dos grafos y enlázalos mediante la propiedad OWL:sameAs, inserta estas coincidencias en g3. Consideramos dos individuos iguales si tienen el mismo apodo y nombre de familia. Ten en cuenta que las URI no tienen por qué ser iguales para un mismo individuo en los dos grafos."""
+
+ns1 = Namespace("http://data.three.org#")
+ns2 = Namespace("http://data.four.org#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+q1 = prepareQuery('''
+  SELECT ?Subject WHERE { 
+    ?class rdfs:subClassOf* ns:Person .
```

Comment:
Here you are not being asked about person, but about individuals with name and last name. You can use that to simplify this

Reply:
You are right, It doesn't say anything about persons. So, do I just need to find all subjects with name and last name, no matter which class they belong to?

Reply:
Perfect, thanks for the correction!!!